waspc/src/Wasp/Analyzer/Parser/PrettyPrinter.hs

```diff
@@ -0,0 +1,16 @@
+module Wasp.Analyzer.Parser.PrettyPrinter
+  ( prettyPrintParserResult,
+  )
+where
+
+import Wasp.Analyzer.Parser hiding (withCtx)
```

Comment:
Why are we hiding `withCtx`?
Also, we should probably be doing a qualified import.

Finally, the best choice is probably `import Wasp.Analyzer.Parser (ParseError, AST)`